CONTRIBUTING.md

```diff
@@ -21,7 +21,7 @@ npm run build
 
 # Start Next.js Demo
 cd packages/demo-next
-npm run dev
+npm run dev-next
```

Comment:
Hey, was this change intentional?

Reply:
Good catch, it could be this or `develop` I forgot we removed the `dev` script

Reply:
Yes, it was intentional, although not strictly related to fixing #1640 😄 
Just noticed the run command was not correct while working on this change

Reply:
Reverted 910f1411632773b70f2078da006e7ad8f2888ff4 to exclude this change

Reply:
Ah, my bad. Thanks!